.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   ubuntu:
     strategy:
       matrix:
-        version: ["PHP-8.0", "master"]
+        version: ["PHP-8.1", "PHP-8.2"]
```

Comment:
I'd still keep the tests for PHP-8.0 (also for Windows below); there are likely users still on this PHP version, and it may even still be supported by some distros.